src/main/java/org/humancellatlas/ingest/state/monitor/SubmissionStateMonitor.java

```diff
@@ -113,32 +111,30 @@ public boolean sendEventForSubmissionEnvelope(SubmissionEnvelopeReference submis
         if (stateMachine.isPresent()) {
             StateMachine<SubmissionState, SubmissionEvent> machine = stateMachine.get();
             return machine.sendEvent(event);
-        }
-        else {
+        } else {
             throw new IllegalArgumentException(String.format(
                     "Submission envelope reference '%s' is not currently being monitored",
                     submissionEnvelopeReference.getUuid()));
         }
     }
 
-    public boolean notifyOfMetadataDocumentState(MetadataDocumentReference metadataDocumentReference, SubmissionEnvelopeReference submissionEnvelopeReference, MetadataDocumentState state){
+    public boolean notifyOfMetadataDocumentState(MetadataDocumentReference metadataDocumentReference, SubmissionEnvelopeReference submissionEnvelopeReference, MetadataDocumentState state) {
         Optional<StateMachine<SubmissionState, SubmissionEvent>> stateMachine =
-            findStateMachine(UUID.fromString(submissionEnvelopeReference.getUuid()));
+                findStateMachine(UUID.fromString(submissionEnvelopeReference.getUuid()));
 
         if (stateMachine.isPresent()) {
```

Comment:
I think this could also use the Optional API. Consider `map()`, `orElseThrow()`.